ui/components/ui/tabs/dropdown-tab/dropdown-tab.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Dropdown from '../../dropdown';
+
+export const DropdownTab = (props) => {
+  const {
+    activeClassName,
+    className,
+    'data-testid': dataTestId,
+    isActive,
+    onClick,
+    onChange,
+    tabIndex,
+    options,
+    selectedOption,
+  } = props;
+
+  return (
+    <li
+      className={classnames('tab', className, {
```

Comment:
I'd vote to rename this class 'dropdown-tab' versus just 'tab'.

Reply:
@georgewrmarshall just pinging for a double check on whether this is something that should be considered for inclusion in design system components. IT seems small enough that it might be.

Reply:
Also should this be in uit/tabs ? Does it *use* the 'tab' component or is it wholly self sufficient?

Reply:
Yup, it does! Have requested stories for all new UI components in this PR so we can easily review and iterate on in future

Reply:
@brad-decker it uses style from the `Tab` component and the [confirm-page-container-content css](https://github.com/MetaMask/metamask-extension/pull/15814/files#diff-37b35ddf3d088e4a3707b8fbc05c06ae8aeba47b1ac77905359865d3760231a8) uses this class to apply some style to the tabs. I can add an additional class tho.